cvcl/forms.py

```diff
@@ -49,6 +47,10 @@ def process_data(self):
         reader = csv.DictReader(f)
 
         for user in reader:
-            email2 = user['Username'] + "@drexel.edu"
-            User.objects.create_user(user['Username'], email=email2, first_name=user['First Name'],
-                                     last_name=user['Last Name'])
+            email = user['Username'] + "@drexel.edu"
+            username = user['Username']
+            if User.objects.filter(username=username).exists():
+                raise forms.ValidationError(u'Username "%s" already exists' % username)
```

Comment:
So you're throwing an error just because the user exists in the database? No way to add him/her to course, huh?